tests/Functional/Language/FileSourceBuilderTest.php

```diff
@@ -0,0 +1,34 @@
+<?php
+
+namespace Digia\GraphQL\Test\Functional\Language;
+
+use Digia\GraphQL\Language\FileSourceBuilder;
+use Digia\GraphQL\Test\TestCase;
+
+/**
+ * Class FileSourceBuilderTest
+ * @package Digia\GraphQL\Test\Functional\Language
+ */
+class FileSourceBuilderTest extends TestCase
+{
+
+    /**
+     * @expectedException \Digia\GraphQL\Error\InvariantException
+     */
+    public function testFileNotFound(): void
+    {
+        $builder = new FileSourceBuilder('/not/existing/file.graphqls');
```

Comment:
I think it would be better if the builder would be stateless. I would consider changing the build method to accept the path(s) instead.

Reply:
Different builders need different arguments, I don't think we can make those a part of the interface?

Reply:
Oh right, let's keep it as is then.